boards/lckfb/esp32c3_lckfb/esp32c3_lckfb-pinctrl.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024 LCKFB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32c3-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32c3-gpio-sigmap.h>
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO21>;
+			output-high;
+		};
+
+		group2 {
+			pinmux = <UART0_RX_GPIO20>;
+			bias-pull-up;
+		};
+	};
+
+	spim2_default: spim2_default {
+		group1 {
+			pinmux = <SPIM2_SCLK_GPIO3>,
+				 <SPIM2_CSEL_GPIO4>;
+		};
+
+		group2 {
+			pinmux = <SPIM2_MOSI_GPIO5>;
+			output-low;
+		};
+	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_GPIO0>,
+				 <I2C0_SCL_GPIO1>;
+			bias-pull-up;
+			drive-open-drain;
+			output-high;
+		};
+	};
+
+	i2s_default: i2s_default {
+		group1 {
+			pinmux = <I2S_MCLK_GPIO10>,
+				 <I2S_O_BCK_GPIO8>,
+				 <I2S_I_SD_GPIO7>;
+		};
+	};
+
+	twai_default: twai_default {
```

Comment:
I guess you should remove this as those pins GPIO4/GPIO5 are attached to SPI/LCD.